association-frontend/hooks/use-user/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { useCallback, useContext } from "react";
+
+import { UserContext } from "~/components/user-provider";
+
+export const useUser = () => {
+  const { user, resetUrqlClient } = useContext(UserContext);
+  const logout = useCallback(() => {
+    resetUrqlClient?.();
+  }, [resetUrqlClient]);
```

Comment:
not for this PR, but we need to better define the logout flow, because right now the Gateway handles everything, so this logout doesn't delete the identity cookies or anything